src/pages/Main/Slider/BottomSlider.js

```diff
@@ -0,0 +1,142 @@
+import React, { useEffect, useRef, useState } from 'react';
+import BottomSlideContents from './BottomSlideContents';
+import styled from 'styled-components';
+
+export default function BottomSlider() {
+  const slideRef = React.useRef();
```

Comment:
```suggestion
export default function BottomSlider() {
  const slideRef = useRef();
```
이미 구조분해 할당으로 useRef를 분해해 놓았는데 굳이 React에서 가져올 필요 없을 것 같습니다

Reply:
해당 부분도 어떤 의미인지 알지 못하고 그냥 썼던것 같습니다. 
import하는 부분에서도 구조분해할당이 된다는 것을 몰랐습니다. 감사합니다~~